tests/src/main/java/org/hibernate/beanvalidation/tck/tests/xmlconfiguration/MissingClassNameOnBeanNodeTest.java

```diff
@@ -34,15 +37,9 @@ public static WebArchive createTestArchive() {
 				.build();
 	}
 
-	@Test
+	@Test(expectedExceptions = ValidationException.class)
 	@SpecAssertion(section = Sections.XML_MAPPING_CONSTRAINTDECLARATIONINXML, id = "c")
 	public void testClassNameIsMandatory() {
-		try {
-			TestUtil.getValidatorUnderTest();
-			fail();
-		}
-		catch ( ValidationException e ) {
-			// success
-		}
+		TestUtil.getValidatorUnderTest();
```

Comment:
Is it on purpose that there's no `fail()` call here as with the others?

Reply:
I removed the `fail()` calls when they were totally useless as it's just an old pattern. They were even counter productive.

I kept them when the message was interesting and conveyed some information as it was better than the error generated by TestNG.

So, yes, it's consistent.